recipe/meta.yaml

```diff
@@ -3,7 +3,7 @@
 # Copyright (c) 2017, Continuum Analytics, Inc. All rights reserved.
 #
 # Distribution of the content approved by NVIDIA ( http://nvbugs/3052604 )
-{% set major_minor = major_minor_ver|default("11.2.2") %}
+{% set major_minor = major_minor_ver|default("11.3.1") %}
```

Comment:
I guess we'll need a follow-up PR to bump to 11.4.2 after this one is done?

Reply:
Any reason to not go to 11.4.2 now?

Reply:
Yea I can't see why not. I am not even sure we are hitting the `default` there though.

Reply:
@jaimergp If we bump this to 11.4, do we still get 11.3 published?!

Reply:
Yes, this is controlled through the CBC:

https://github.com/conda-forge/cudatoolkit-feedstock/blob/52b09a476f7a26c1c363217c9179a8d0e75dd01d/recipe/conda_build_config.yaml#L4-L13

Reply:
ooohhh sorry I was being silly, thanks Jaime.